distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestNsili.java

```diff
@@ -207,10 +214,11 @@ private void assertMetacards(String responseXml, int expectedNumMetacards,
 
         public static final String FACTORY_PID = "NSILI_Federated_Source";
 
-        public NsiliSourceProperties(String sourceId, String iorUrl) {
+        public NsiliSourceProperties(String sourceId, String httpIorUrl, String ftpIorUrl) {
             this.putAll(getMetatypeDefaults(SYMBOLIC_NAME, FACTORY_PID));
             this.put("id", sourceId);
-            this.put("iorUrl", iorUrl);
+            this.put("iorUrl", httpIorUrl);
+            this.put("ftpIorUrl", ftpIorUrl);
```

Comment:
Unless i'm misunderstanding the intention here, the url property should correspond to the metatype id, which is the iorUrl.  I think we will need two different NSILI Source configurations (one web and one ftp) and test them individually.